qutebrowser/config/configfiles.py

```diff
@@ -566,12 +566,16 @@ def _handle_error(self, action: str, name: str) -> typing.Iterator[None]:
 
     def finalize(self) -> None:
         """Do work which needs to be done after reading config.py."""
+        with self._handle_error('loading', 'autoconfig'):
+            if config.instance.warn_autoconfig:
+                raise configexc.Error(
+                        "The autoconfig hasn't been specified to be loaded or not")
         self._config.update_mutables()
 
-    def load_autoconfig(self) -> None:
+    def load_autoconfig(self, load_config: bool = True) -> None:
         """Load the autoconfig.yml file which is used for :set/:bind/etc."""
         with self._handle_error('reading', 'autoconfig.yml'):
-            read_autoconfig()
+            read_autoconfig(load_config)
```

Comment:
Wouldn't it be simpler to set `config.instance.warn_autoconfig` in here and not call `read_autoconfig` at all if `load_config` is set to `False`?

Reply:
yeah, it would be much simpler, I will change